mock/el7-scl.cfg

```diff
@@ -6,6 +6,7 @@ config_opts['dist'] = 'el7'  # only useful for --resultdir variable subst
 # beware RHEL use 7Server or 7Client
 config_opts['releasever'] = '7'
 config_opts['package_manager'] = 'yum'
+config_opts['yum_install_command'] += ' --disablerepo sclo*'
```

Comment:
Mind separating this to a separate commit? https://github.com/theforeman/foreman-packaging/pull/5047 was intended to solve the same issue

Reply:
This is not the final commit. I just kept things in here I need present to allow me to test and re-test.